.github/workflows/release.yaml

```
@@ -0,0 +1,32 @@
+name: Upload acme.js to release page
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    name: Build release artifact
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.3.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Build dist/acme.js
+        run: make build
+
+      - name: Upload acme.js to release
+        uses: actions/upload-release-asset@v1.0.2
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          upload_url: ${{ steps.get_release.outputs.upload_url }}
```

I'm not sure we need `bruceadams`; it's a few years without changes and the [release event](https://docs.github.com/en/webhooks/webhook-events-and-payloads#release) looks like it has the data we need.

I think we can get that from the [github context](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context):

```suggestion
      - name: Build dist/acme.js
        run: make build

      - name: Upload acme.js to release
        uses: actions/upload-release-asset@v1.0.2
        env:
          GITHUB_TOKEN: ${{ github.token }}
        with:
          upload_url: ${{ github.event.release.upload_url }}
```

This stuff is really hard to test; I end up making a throwaway repo to try things out and see what happens, using a  `run: echo "${{ toJson(github) }}"` step to show me what is actually there. This suggestion is irrelevant if we switch to using `gh release upload`.